composer.json

```diff
@@ -6,8 +6,9 @@
   "require": {
     "php": ">=5.6",
     "composer/installers": "^v1.12.0 || ^2.2",
-    "dealerdirect/phpcodesniffer-composer-installer": "^1.0.0",
-    "wp-coding-standards/wpcs": "dev-develop#fca9d9e"
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7 || ^1.0",
+    "wp-coding-standards/wpcs": "^2.3 | dev-develop#fca9d9e",
```

Comment:
This changes introduce #136 issue again.

Reply:
@jjgrainger Can you explain why this is needed?

Reply:
Here is the fix https://github.com/10up/plugin-check/pull/193/files#r1235184078

Reply:
Thanks @spacedmonkey the alias fixes the #136 regression, however I'm seeing a new issue.

Seems there is still some issues with dependencies as now this performant query check no longer works. When testing this directly with PHPCS I see a new `ERROR: Referenced sniff "WordPress.PHP.StrictComparisons" does not exist`.

It looks some sniffs have been moved but not referenced correctly by the VIP package. I'm still investigating...

cc @mukeshpanchal27

Reply:
I can't replicate. Can you push up your branch @jjgrainger I can look at what you are seeing.

Reply:
Not related, but found in my search https://github.com/WordPress/WordPress-Coding-Standards/blob/f2b63051f08cf6d51360896bb1647eabc6911bf2/CHANGELOG.md?plain=1#L143

Reply:
This is my branch. https://github.com/10up/plugin-check/pull/196

Reply:
Thanks @spacedmonkey 

Your solution works in order to pull in the correct dependencies, however I'm facing a new issue after this.

When I run the following command on my branch to make sure that PHPCS sniffs are working with the test plugin I get the follow error.

```
./vendor/bin/phpcs --standard="WordPress,WordPressVIPMinimum" --sniffs="WordPress.DB.SlowDBQuery,WordPressVIPMinimum.Performance.WPQueryParams" ./tests/phpunit/testdata/plugins/test-plugin-performant-wp-query-params-errors/load.php
```

```
ERROR: Referenced sniff "WordPress.PHP.StrictComparisons" does not exist
```

I believe this is to do with wpcs moving certain sniffs to a different name but the vipcs references the old names. I've tried to see if there is a combination of versions that will 1) resolve this issue and 2) solve the https://github.com/10up/plugin-check/issues/136 regression. However, I've had no luck.

As it looks like we can't get to an ideal solution, I believe the best option to move forward quickly is to remove the vipcs package and have this check only run the `WordPress.DB.SlowDBQuery` sniff for now.

We can then create a new issue to add the `WordPressVIPMinimum.Performance.WPQueryParams` sniff back in once both vipcs and wpcs packages have been updated and we can use compatible stable versions that do no reintroduce the issue with #136 

I'd like to hear your thoughts on this @mukeshpanchal27 @felixarntz @spacedmonkey and if agreed we can make those updates and finalise this branch.

Thanks!

Reply:
@jjgrainger We discuss this here: https://github.com/10up/plugin-check/pull/196#discussion_r1236471117